web/backend/routes/root.ts

```diff
@@ -29,13 +29,23 @@ export default async function (fastify: FastifyInstance) {
       },
     }
   }, async (request) => {
-    let runtimes = await api.getRuntimes()
+    const runtimes = await api.getRuntimes()
+
+    type SelectableRuntime = typeof runtimes[0] & { selected?: boolean }
+
+    let selectableRuntimes: SelectableRuntime[] = runtimes
 
     if (!request.query.includeAdmin) {
-      runtimes = runtimes.filter((runtime) => runtime.packageName !== 'watt-admin')
+      selectableRuntimes = selectableRuntimes.filter((runtime) => runtime.packageName !== 'watt-admin')
+    }
+
+    for (const runtime of selectableRuntimes) {
+      if (process.env.SELECTED_RUNTIME === runtime.pid.toString()) {
```

Comment:
If I understand properly, we allow the user from the `cli.js` to only select one runtime at a time.
Since this is the case, I think it makes more sense to return directly the runtime (rather than an array of runtimes).

In case `watt-admin` is started outside from the `./cli.js` tool, we can rather select the first runtime (if multiple ones are started). WDYT?